test/celeritas/em/Fluctuation.test.cc

```diff
@@ -271,16 +271,17 @@ TEST_F(EnergyLossDistributionTest, urban)
         counts[bin]++;
         sum += loss;
     }
-
-    if (CELERITAS_REAL_TYPE == CELERITAS_REAL_TYPE_DOUBLE)
-    {
-        static real_type const expected_counts[]
-            = {0,   0,   12, 223, 1174, 2359, 2661, 1867, 898, 369,
-               189, 107, 60, 48,  20,   9,    2,    2,    0,   0};
-        EXPECT_VEC_SOFT_EQ(expected_counts, counts);
-        EXPECT_SOFT_EQ(0.0099918954960280353, sum / num_samples);
-        EXPECT_EQ(551188, rng.count());
-    }
+#ifdef _MSC_VER
+    // XXX
```

Comment:
Is this missing an update/commit?

Reply:
Ah yes I should actually write why this is disabled, thanks!

Reply:
Oh right, I wrote it in the "skip" message. But I'll change this to a "TODO" and add a comment.